src/DynamoSandbox/Program.cs

```diff
@@ -154,6 +104,27 @@ internal class Program
     {
         private static SettingsMigrationWindow migrationWindow;
 
+        private static DynamoModel makeModel()
```

Comment:
Always use `PascalCase` when writing .NET code:

https://msdn.microsoft.com/en-us/library/x2dbyw72(v=vs.71).aspx

Also, don't create duplicate code.  It looks like your method could be used in `MakeStandaloneAndRun`.